x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/plotUtils.js

```diff
@@ -71,7 +71,9 @@ export function getPlotValues(
   const yScale = getYScale(yMin, yMax);
 
   const yMaxNice = yScale.domain()[1];
-  const yTickValues = [0, yMaxNice / 2, yMaxNice];
+  const yTickValues = !isNaN(yMaxNice)
+    ? [0, yMaxNice / 2, yMaxNice]
+    : [0, 0.5, 1];
```

Comment:
This change doesn't look right. If `yMaxNice` is `NaN` we'll have other problems too.

Reply:
it's no longer necessary if we pass an empty series with `1` as an 'empty' value rather than `null`. I do think we're adding hack on hack here, but maybe I have not considered all possible requirements yet.

Reply:
How will the new `noHits` check work when you are passing in `1`?

Reply:
we only use 1 internally (in CustomPlot). Let me see how it would look.